exercises/pangram/example.rs

```diff
@@ -0,0 +1,18 @@
+use std::collections::BTreeSet;
+use std::ascii::AsciiExt;
+use std::iter::FromIterator;
+
+pub fn is_pangram(sentence: &str) -> bool {
+    sentence.to_lowercase()
+            .chars()
```

Comment:
@IanWhitney haha, that shows how long it's been since I've played in Rust.  Code looks good to me, but now I am not so confident in my Rust review skills haha.